Test/git-issues/git-issue-3978.dfy

```diff
@@ -0,0 +1,13 @@
+// RUN: %testDafnyForEachCompiler "%s"
```

Comment:
@robin-aws  your script required me to change my test for the Go compiler to every compiler. Nice job!